src/tabs/default.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  @include iui-tabs;
+
+  > li {
+    > button {
```

Comment:
This can be flattened to reduce specificity (by adding a new classname)

Reply:
Flatted by adding `.iui-tab` class to all `<button>`s.